pkgs/by-name/ne/netbird/profilemanager-panic-fix.patch

```diff
@@ -0,0 +1,84 @@
+From 7320684dd1893d8edc731f9d3ef71579b6e40869 Mon Sep 17 00:00:00 2001
+From: Krzysztof Nazarewski <gpg@kdn.im>
+Date: Fri, 8 Aug 2025 17:21:02 +0200
+Subject: [PATCH] fix: profilemanager panic when reading incomplete config
+
+Aug 08 15:09:08 clients systemd[1]: Started A WireGuard-based mesh network that connects your devices into a single private network.
+Aug 08 15:09:09 clients netbird[1599]: 2025-08-08T15:09:09Z INFO client/cmd/service_controller.go:27: starting Netbird service
+Aug 08 15:09:09 clients netbird[1599]: panic: runtime error: invalid memory address or nil pointer dereference
+Aug 08 15:09:09 clients netbird[1599]: [signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x60f56c]
+Aug 08 15:09:09 clients netbird[1599]: goroutine 13 [running]:
+Aug 08 15:09:09 clients netbird[1599]: net/url.(*URL).String(0x2e117a0?)
+Aug 08 15:09:09 clients netbird[1599]:         net/url/url.go:833 +0x2c
+Aug 08 15:09:09 clients netbird[1599]: github.com/netbirdio/netbird/client/server.(*Server).Start(0xc00034ac00)
+Aug 08 15:09:09 clients netbird[1599]:         github.com/netbirdio/netbird/client/server/server.go:159 +0x675
+Aug 08 15:09:09 clients netbird[1599]: github.com/netbirdio/netbird/client/cmd.(*program).Start.func1()
+Aug 08 15:09:09 clients netbird[1599]:         github.com/netbirdio/netbird/client/cmd/service_controller.go:65 +0x4e6
+Aug 08 15:09:09 clients netbird[1599]: created by github.com/netbirdio/netbird/client/cmd.(*program).Start in goroutine 1
+Aug 08 15:09:09 clients netbird[1599]:         github.com/netbirdio/netbird/client/cmd/service_controller.go:54 +0x2be
+Aug 08 15:09:09 clients systemd[1]: netbird.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
+Aug 08 15:09:09 clients systemd[1]: netbird.service: Failed with result 'exit-code'.
+---
+ client/cmd/service.go                    |  2 +-
+ client/internal/profilemanager/config.go | 19 +++++++++----------
+ 2 files changed, 10 insertions(+), 11 deletions(-)
+
+diff --git a/client/cmd/service.go b/client/cmd/service.go
+index 0c566370b09..997520f4cff 100644
+--- a/client/cmd/service.go
++++ b/client/cmd/service.go
+@@ -47,7 +47,7 @@ func init() {
+ 	rootCmd.PersistentFlags().StringVarP(&serviceName, "service", "s", defaultServiceName, "Netbird system service name")
+ 	serviceEnvDesc := `Sets extra environment variables for the service. ` +
+ 		`You can specify a comma-separated list of KEY=VALUE pairs. ` +
+-		`E.g. --service-env LOG_LEVEL=debug,CUSTOM_VAR=value`
++		`E.g. --service-env NB_LOG_LEVEL=debug,CUSTOM_VAR=value`
```

Comment:
Is that a mistake, or are you renaming the LOG_LEVEL variable deliberately? It doesn't seem related to the patches purpose on first glance?

Reply:
it indeed doesn't, must have commited the cosmetic change with the fix